.github/workflows/on_build_do_test.yml

```diff
@@ -102,7 +111,7 @@ jobs:
             test_dir/infinispan/**/hs_err_*
             github-sha.txt
             !test_dir/infinispan/**/*-reports*/**/TEST-*FLAKY.xml
-            !test_dir/infinispan/**/*-reports*/TEST-*RocksDBReaderTest*.xml
+            !test_dir/infinispan/**/*-reports*/TEST-*RocksDB*.xml
```

Comment:
Any idea why the RocksDB output is malformed? It would be good to create a follow up issue to fix this.

Reply:
no idea, I'll truck it in a jira